webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1348,16 +1348,15 @@ function callbackElement(oData, displayCompliance) {
 
 
 var dynColumns = []
-var columns = []// allColumns["Hostname"],  allColumns["OS"],  allColumns["Compliance"],  allColumns["Last run"]];
 var defaultColumns = [ allColumns["Score"], allColumns["Hostname"],  allColumns["OS"],  allColumns["Policy mode"],  allColumns["Compliance"]];
 var allColumnsKeys =  Object.keys(allColumns)
-function reloadTable(gridId) {
+async function reloadTable(gridId) {
   var table = $('#'+gridId).DataTable();
   table.destroy();
-  createNodeTable(gridId, function(){reloadTable(gridId)})
+  await createNodeTable(gridId, function(){reloadTable(gridId)})
 }
```

Comment:
Rewriting to async/await is not necessary here but still a good practice since later we would be able to "await" the reload of the table